submissions/UA-girl/task_js_oop_exersise/index.js

```diff
@@ -0,0 +1,55 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/UA-girl/a-tiny-JS-world
+   Web app: https://ua-girl.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+class Creature {
+    constructor(species, name, age, gender, legs, say, friends = []) {
+        this.species = species;
+        this.name = name;
+        this.age = age;
+        this.gender = gender;
+        this.say = say;
+        this.legs = legs
+        this.friends = friends.length > 0 ? friends : [];
+    }
+
+    toString() {
+        return `${this.say}! My name is ${this.name} and I am ${this.age} years old ${this.gender} ${this.species}.`
+    }
+}
+
+class Person extends Creature {
+    constructor(species, name, age, gender, legs, say, friends) {
+        super(species, name, age, gender, legs, say, friends);
+        this.hands = 2;
+    }
+
+    toString() {
+        return super.toString() + ` I have ${this.legs} legs and ${this.hands} hands. My friends are ${this.friends.join(', ')}.`
```

Comment:
Why shared properties like species, gender, name, age and say are handled in the base class `toString`, while legs and friends are not?

Note how similar this question to the one I had at the previous iteration of the review.

Please induce and deduce new information from explicit statements like this.

Reply:
I've made this by design for my simplicity to form a message (Animals have no hand), but you right, I've found better way for this. In fact, method toString can be stored in the parent class only.